app/views/pages/use-cases/twilio-video-integration.liquid

```diff
@@ -0,0 +1,302 @@
+---
+converter: markdown
+slug: use-cases/twilio-video-integration
+metadata:
+  title: Setting Up Twilio Video Integration
+  description: Setup Twilio video integration for allowing users to engage in video conferencing.
+  tags: ["context", "parse_json", "jwt_encode", "form_configuration", "api_call_notifications"]
+---
+
+This use case describes how we created a video conferencing integration for **platformOS** using **[Twilio](https://www.twilio.com)**. 
+
+To create the integration, first we needed to connect platformOS and Twilio. There are many tools available to connect to Twilio and get the API Keys to generate the JWT tokens needed for Twilio's video services. As we would need to interact with the Twilio API on user events later in our module and the best and most efficient way to do this would be directly from platformOS, we decided to learn the ropes and use all platformOS had to offer from the start.  
+
+## Problem/situation
+
+First, we needed to get and store all of the API Keys necessary to authorize the connection. We could get some of them from the platformOS Partner Portal after adding the Twilio integration as detailed in the topic [Managing Integrations](https://documentation.platformos.com/tutorials/partner-portal/instances/managing-integrations). To use other API resources, we created a form and listened to pOS logs in our CLI. Then we created a JWT (JSON Web Token) that would allow the connection to take place. 
```

Comment:
```suggestion
First, we needed to get and store all of the API Keys necessary to authorize the connection. We could get some of them from the platformOS Partner Portal after adding the Twilio integration as detailed in the topic [Managing Integrations](/tutorials/partner-portal/instances/managing-integrations). To use other API resources, we created a form and listened to pOS logs in our CLI. Then we created a JWT (JSON Web Token) that would allow the connection to take place. 
```